multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java

```diff
@@ -32,41 +32,14 @@
 
     @Inject
     @Bean
-    public DynatraceMeterRegistry dynatraceMeterRegistry(ApplicationConfiguration appConfig) {
+    public JmxMeterRegistry jmxMeterRegistry(ApplicationConfiguration appConfig) {
         DynatraceServiceInfo serviceInfo = getDynatraceServiceInfo();
```

Comment:
This should no longer be needed, right?

Reply:
DynatraceMeterRegistry  is removed. ReactorNetty use directly Metrics.globalRegistry that propagates meters in all registered registries.